sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 2.0.4 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 2.0.4 (2022-02-18)
 
 ### Bugs Fixed
 
-### Other Changes
+- 2.0.3 contains a regression: Passing a client Id besides the options argument on the `ManagedIdentityCredential` would discard the options argument. We have fixed it and we have added tests.
```

Comment:
```suggestion
- Fixed a regression in version 2.0.3 in which providing an options bag, but _not_ a client ID, to the `ManagedIdentityCredential` constructor would discard the `options` parameter.
```

I think this description of the fix is easier to understand based on the code changes I read below.

Reply:
No no, the bug happens when you provide a `clientId` _and_ the options bag.

Reply:
You’re right!